examples/records_with_no_created_at_timestamp/connector.py

```diff
@@ -0,0 +1,91 @@
+# This is an example to show how to work with records where the source does not provide a created_at(or equivalent) field.
+# If you want to keep track of when the record was first observed, then the updated_at(or equivalent, eg, modified_at etc)
+# field can be made part of the composite Primary Key. By doing this you can ensure that successive syncs do not rewrite the
+# updated_at field in the existing row in the destination. Instead, it creates a new row with the new updated_at.
+# Now, the record with earliest value of _fivetran_synced system column represents when the record was first observed in source.
+# Refer to the Resulting table section at the bottom of this file for a detailed demonstration.
+# See the Technical Reference documentation (https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update)
+# and the Best Practices documentation (https://fivetran.com/docs/connectors/connector-sdk/best-practices) for details
+
+# Import required classes from fivetran_connector_sdk
+from fivetran_connector_sdk import Connector
+from fivetran_connector_sdk import Operations as op
+
+
+def schema(configuration: dict):
+    return [
+        {
+            "table": "user",  # Name of the table in the destination.
+            "primary_key": ["id", "updated_at"],  # Primary key column(s) for the table.
+            "columns": {  # Define the columns and their data types.
+                "id": "INT",
+                "first_name": "STRING",  # String column for the first name.
+                "last_name": "STRING",  # String column for the last name.
+                "updated_at": "UTC_DATETIME",  # UTC date-time column for the updated_at.
+            },
+        }
+    ]
+
+
+# Define the update function, which is a required function, and is called by Fivetran during each sync.
+# See the technical reference documentation for more details on the update function
+# https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update
+# The function takes two parameters:
+# - configuration: dictionary contains any secrets or payloads you configure when deploying the connector
+# - state: a dictionary contains whatever state you have chosen to checkpoint during the prior sync
+# The state dictionary is empty for the first sync or for any full re-sync
+def update(configuration: dict, state: dict):
+    # Represents a record fetched from source
+    row_1 = {
+        "id": 123,
+        "first_name": "John",  # First name.
+        "last_name": "Doe",  # Last name.
+        "designation": "Manager",  # Designation
+        "updated_at": "2007-12-03T10:15:30Z"  # Updated at timestamp.
```

Comment:
I am not understanding this part clearly, are you assuming that we will be getting this `updated_at` column from the source? If that is the case, then yes having it as part of PK should solve the problem. 

But what if the customer table doesn't have any columns like `created_at` and `updated_at`, should we also cover this scenario, what customer can do in this case? 
E.g. we can add the `updated_at` column from the code (the timestamp would be during sync_time) and this would also help them identify which row was synced at what time.

Reply:
> are you assuming that we will be getting this updated_at column from the source
Yes, the assumption here is that the source will provide an `updated_at` (or equivalent - `modified_at` etc) column. 

`But what if the customer table doesn't have any columns like created_at and updated_at`
It was decided that we do not want to handle this case through code.  IMO the `_fivetran_synced` system column should suffice in this scenario. 

`we can add the updated_at column from the code (the timestamp would be during sync_time) and this would also help them identify which row was synced at what time.`
It was discussed that we should not be doing this as this ain't good practice and the `_fivetran_synced` system column should suffice.

Reply:
Ohhk, got it.